core/server/data/import/utils.js

```diff
@@ -234,6 +234,11 @@ utils = {
                 post.created_at = Date.now();
             }
 
+            _.each(['created_at', 'updated_at', 'published_at'], function (fieldName) {
```

Comment:
Is there a reason why you only sanitise the dates for posts?
Exotic date formats can be present for all model types e.g. users.
How about putting this piece of code to `import/index.js sanitize fn`?